tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -561,6 +563,68 @@ public function testAlterTableAutoIncrementIntToBigInt(
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testListTableColumnsOidConflictWithNonTableObject(): void
+    {
+        if ($this->connection->getDatabasePlatform() instanceof PostgreSQL94Platform) {
```

Comment:
This will skip the test on all Postgres versions because…

https://github.com/doctrine/dbal/blob/82483595da680180c1d360ea7fbff1ea8fb08142/src/Platforms/PostgreSQL100Platform.php#L16